content/docs/components/textarea/theming.mdx

```diff
@@ -2,3 +2,37 @@
 id: textarea
 scope: theming
 ---
+
+## Overview
+
+The `Textarea` component is a single part component, To learn more about
+singlepart components, visit the
+[Component Style](/docs/styled-system/component-style#styling-single-part-components)
+page.
+
+## Defaults
+
+- The default `variant` is `outline`
+- The default `size` is `md`
+
+## Theming
+
+import {
+  App,
+  Index,
+  TextareaTheme,
+} from 'configs/sandpack-contents/component-theming/textarea'
+
+<SandpackEmbed
+  files={{
+    '/theme/components/Button.ts': TextareaTheme,
```

Comment:
Your `Button.ts` should actually be `Textarea.ts`.

Reply:
ohh k

Reply:
> Your `Button.ts` should actually be `Textarea.ts`.

working

Reply:
![Screenshot 2022-10-12 at 1 59 45 AM](https://user-images.githubusercontent.com/85970385/195192330-9fa660fd-a2df-4d2c-a112-a2019f81c7b8.png)
![Screenshot 2022-10-12 at 1 59 59 AM](https://user-images.githubusercontent.com/85970385/195192362-565ef1d2-8d68-440e-8247-508729215224.png)
![Screenshot 2022-10-12 at 2 00 11 AM](https://user-images.githubusercontent.com/85970385/195192392-783c0fd4-f730-4863-bc9f-cc2f3623e33f.png)